src/org/linphone/VideoCallFragment.java

```diff
@@ -62,8 +62,31 @@
 	@SuppressWarnings("deprecation") // Warning useless because value is ignored and automatically set by new APIs.
 	@Override
 	public View onCreateView(LayoutInflater inflater, ViewGroup container, 
-			Bundle savedInstanceState) {		
-		View view = inflater.inflate(R.layout.video, container, false);
+			Bundle savedInstanceState) {
+
+		int viewId = R.layout.video;
+		LinphoneCallParams params;
+		LinphoneCall call;
+
+		try {
+			call = LinphoneManager.getLc().getCurrentCall();
+			params = call.getCurrentParamsCopy();
+			if (params.getUsedVideoCodec().toString().contains("H263")) {
+
+				LinphoneManager.getLc().setDeviceRotation(90);
+				LinphoneManager.getLc().updateCall(call, null);
+				viewId = R.layout.video_h263;
+
+			}
+		}
+
+		catch(NullPointerException e){
```

Comment:
My bad! This was at the end of the day and we just wanted something to work. I'll add some checks and remove the try / catch right now.